src/metabase/models/params/chain_filter.clj

```diff
@@ -115,26 +115,21 @@
 
 (defn- filter-clause
   "Generate a single MBQL `:filter` clause for a Field and `value` (or multiple values, if `value` is a collection)."
-  [source-table-id field-id value]
+  [source-table-id {:keys [field-id op value options]
+                    :or   {op :=}}]
```

Comment:
I don't think op need a default at this level right?

Reply:
Well... I've put it as optional in schema, since there are other users for `chain-filter`, like [filterable-field-ids](https://github.com/metabase/metabase/blob/34039-fix-is-not-filter/src/metabase/models/params/chain_filter.clj#L698)

Reply:
how about we specify the `:op` there other than making an assumption here?

Reply:
yeah why not, just wanted to keep changes to minimum :)

Reply:
done in 39ec5a256ca0cd926f3cbad9232029374e1191fe